packages/cli/src/commands/app.ts

```diff
@@ -1,210 +1,108 @@
-/* eslint-disable prefer-const */
+import { Command, Program } from "./types";
 import {
   AppProject,
   CodeGenerator,
   SchemaComposer,
   intlMsg,
-  fixParameters,
-  resolvePathIfExists,
-  defaultAppManifest,
   getSimpleClient,
   getTestEnvProviders,
 } from "../lib";
+import {
+  parseAppManifestFileOption,
+  parseAppCodegenDirOption,
+  // defaultAppCodegenDirOption,
```

Comment:
What's up with the comments?

Reply:
That's for fixing it back when commander default arg-parser start working.

Reply:
I can remove it though for now

Reply:
Or just add "//TODO: ..."
Or better yet: "//TODO: <issue number>"